### PR TITLE
Update e2e test golden files, along with BT/BQ paths.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: golden_response/staging/get_pop_obs/moscow.json [hunk 1]
```diff
@@ -5,6 +5,286 @@
...
       "observations": [
+        {
+          "id": "dc/o/zxgjjf5jxdmp",
+          "type": "Observation",
+          "measuredProp": "count",
+          "measuredValue": 188700,
+          "measurementMethod": "WikidataPopulation",
+          "provenanceId": "dc/6p25cm",
+          "observationDate": "1785",
+          "statVarIds": ["TotalPopulation"]
```
Comment (tjann) [inline golden_response/staging/get_pop_obs/moscow.json:16]: Do you know why Count_Person is not here?

File: golden_response/staging/get_triples/place.json [hunk 2]
```diff
@@ -242,7 +242,7 @@
       "objectId": "zip/00603",
       "objectName": "00603",
       "objectTypes": ["CensusZipCodeTabulationArea"],
-      "provenanceId": "dc/sm3m2w3"
+      "provenanceId": "dc/5n63hr1"
```
Comment (shifucun) [inline golden_response/staging/get_triples/place.json:245]: @tjann can you check all these provenance change are okay?

File: golden_response/staging/get_stats/NYTCovid19CumulativeCases.json [hunk 3]
```diff
@@ -394,7 +468,44 @@
...
+      "2020-06-28": 4162,
+      "2020-06-29": 4162,
+      "2020-06-30": 4231,
+      "2020-07-01": 4572,
+      "2020-07-02": 4750,
+      "2020-07-03": 4849,
+      "2020-07-04": 5077,
+      "2020-07-05": 5273,
+      "2020-07-06": 5408,
+      "2020-07-07": 5478
```
Comment (shifucun) [inline golden_response/staging/get_stats/NYTCovid19CumulativeCases.json:508]: As I have use the new cache with branch cache capability. Can freeze the base cache for covid data. Then the data here should get pretty stable as well.
@tjann

File: golden_response/staging/get_triples/place_type.json [hunk 4]
```diff
@@ -28,7 +28,7 @@
       "objectId": "Country",
       "objectName": "Country",
       "objectTypes": ["Class"],
-      "provenanceId": "dc/5j06ly1"
+      "provenanceId": "dc/5n63hr1"
```
Comment (shifucun) [inline golden_response/staging/get_triples/place_type.json:31]: @tjann provenance change here

File: golden_response/staging/get_triples/stats_var.json [hunk 5]
```diff
@@ -32,14 +32,6 @@
       "objectTypes": ["Class"],
       "provenanceId": "dc/5l5zxr1"
     },
-    {
-      "subjectId": "MarriedPopulation",
```
Comment (shifucun) [inline golden_response/staging/get_triples/stats_var.json:36]: can you update the stats var in get_triples_test.go to use the new stats var name: "Count_Person", "Count_Person_Female"

File: golden_response/staging/get_pop_obs/moscow.json (hunk 1 above)
Comment (Spaceenter) [reply golden_response/staging/get_pop_obs/moscow.json:16]: Now we have it - changed the stat var DCID in the test.

File: golden_response/staging/get_triples/stats_var.json (hunk 5 above)
Comment (Spaceenter) [reply golden_response/staging/get_triples/stats_var.json:36]: Done.

File: golden_response/staging/get_triples/place_type.json (hunk 4 above)
Comment (tjann) [reply golden_response/staging/get_triples/place_type.json:31]: Went from WikidataGeos to WikidataOtherIdGeos. Seems reasonable for now.

File: golden_response/staging/get_stats/NYTCovid19CumulativeCases.json (hunk 3 above)
Comment (tjann) [reply golden_response/staging/get_stats/NYTCovid19CumulativeCases.json:508]: Ok, sounds good.

**Pull request merged**